src/veh_interact.cpp

```diff
@@ -612,9 +640,12 @@ bool veh_interact::can_install_part() {
     item base( sel_vpart_info->item );
     if( base.is_wheel() ) {
         qual = JACK;
-        lvl = jack_qality( *veh );
+        lvl = jack_quality( *veh );
         str = veh->lift_strength();
-        use_aid = max_jack >= lvl;
+
+        const bool can_self_jack = veh_interact::can_self_jack( veh );
```

Comment:
you shouldn't need to specify a name space for a class function.
alternately, can_self_jack should be a static function outside the veh_interact class, in which case you still shouldn't need the name space.